generator/templates/Authorisation/src/components/authorisation/LoginCard.vue

```diff
@@ -10,43 +10,58 @@
         >
           {{ errorMessage }}
         </v-alert>
-        <VTextField
-            :rules="[(v) => !!v || 'E-Mail is verplicht']"
-            label="E-Mail"
-            tabindex="1"
-            v-model="form.email"
-        />
-        <VTextField
-            :append-icon="showPassword ? 'fa-eye-slash' : 'fa-eye'"
-            :rules="[(v) => !!v || 'Wachtwoord is verplicht']"
-            :type="showPassword ? 'text' : 'password'"
-            @click:append="showPassword = !showPassword"
-            label="Wachtwoord"
-            tabindex="2"
-            v-model="form.password"
-        />
+        <k-field-group language-prefix="authorisation.fields">
+          <KTextField field="email"
+                      autofocus
+                      tabindex="1"
+                      v-model="form.email"
+                      autocomplete="username"
+          />
+          <KTextField :append-icon="showPassword ? 'fa-eye-slash' : 'fa-eye'"
+                      :type="showPassword ? 'text' : 'password'"
+                      @click:append="showPassword = !showPassword"
+                      field="password"
+                      autocomplete="current-password"
+                      tabindex="2"
+                      v-model="form.password"
+          />
+        </k-field-group>
       </v-card-text>
       <v-card-actions>
         <VSpacer/>
-        <v-btn :to="{name: 'password.forgotten'}" color="primary" tabindex="4" text>Wachtwoord vergeten</v-btn>
-        <v-btn :loading="isLoading" color="primary" tabindex="3" type="submit">Inloggen</v-btn>
+        <VBtn :to="{name: 'password.forgotten'}"
+              color="primary"
+              tabindex="4"
+              text
+              v-text="$t('authorisation.actions.passwordForgotten')"/>
+        <VBtn :loading="isLoading"
+              color="primary"
+              tabindex="3"
+              type="submit"
+              v-text="$t('authorisation.actions.login')"/>
       </v-card-actions>
     </v-form>
 
     <v-overlay class="text-center" v-model="isRedirecting">
       <VProgressCircular indeterminate size="64"/>
-      <div class="mt-5">Je bent ingelogd, we sturen je nu door.</div>
+      <div class="mt-5" v-text="$t('authorisation.login.successMessage')"></div>
     </v-overlay>
   </v-card>
 </template>
 
 <script>
+import { getRateLimitMinutes } from '@/api/util/response';
```

Comment:
Ik zou met dit bestand de `BaseForm` of `Form` extenden.

Reply:
Ik heb de baseform juist weggehaald omdat hier eerst eigenlijk alleen het valideren en de servererrors in stonden. Nu dit allen in de KTextField staat is dat niet nodig. Wel moet ik even het valideren idd nakijken